specs/protocol/README.md

```diff
@@ -422,25 +419,38 @@ Currency amounts have type `DecimalString`, which is string containing a decimal
+|----------------|--------------------------------------------------|----------|----------------------------------------------------------------------------|
+| `offeringId`   | string                                           | Y        | Offering which Alice would like to get a quote for                         |
+| `claimsHashes` | string[]                                         | Y        | an array of salted digests of the claims appearing in `privateDate.claims` |
+| `payin`        | [`SelectedPayinMethod`](#selectedpaymentmethod)  | Y        | selected payin amount, method, and details                                 |
+| `payout`       | [`SelectedPayoutMethod`](#selectedpaymentmethod) | Y        | selected payout method, and details                                        |
 
 #### `SelectedPayinMethod`
-| field            | data type                         | required | description                                                                                       |
-| ---------------- | --------------------------------- | -------- | ------------------------------------------------------------------------------------------------- |
-| `amount`         | [`DecimalString`](#decimalstring) | Y        | Amount of payin currency you want in exchange for payout currency                                 |
-| `kind`           | string                            | Y        | Type of payment method (i.e. `DEBIT_CARD`, `BITCOIN_ADDRESS`, `SQUARE_PAY`)                       |
-| `paymentDetails` | object                            | N        | An object containing the properties defined in an Offering's `requiredPaymentDetails` json schema |
+| field                | data type                         | required | description                                                                 |
+|----------------------|-----------------------------------|----------|-----------------------------------------------------------------------------|
+| `amount`             | [`DecimalString`](#decimalstring) | Y        | Amount of payin currency you want in exchange for payout currency           |
+| `kind`               | string                            | Y        | Type of payment method (i.e. `DEBIT_CARD`, `BITCOIN_ADDRESS`, `SQUARE_PAY`) |
+| `paymentDetailsHash` | string                            | N        | A salted digest of `privateData.payin.paymentDetails`                       |
 
 #### `SelectedPayoutMethod`
+| field                | data type | required | description                                                                 |
+|----------------------|-----------|----------|-----------------------------------------------------------------------------|
+| `kind`               | string    | Y        | Type of payment method (i.e. `DEBIT_CARD`, `BITCOIN_ADDRESS`, `SQUARE_PAY`) |
+| `paymentDetailsHash` | object    | N        | A salted digest of `privateData.payout.paymentDetails`                       |
```

Comment:
```suggestion
| `paymentDetailsHash` | object    | N        | A salted hash of `privateData.payout.paymentDetails`                       |
```

"salted digest" is technically correct (I think?) but we use "salted hash" elsewhere so consistency, and my understanding is "salted hash" is the common term to refer to this data type

so anyways, not opposed to "salted digest" but just calling it out because I see it

Reply:
Updated to "salted hash"